### PR TITLE
match AOU skin files

### PR DESCRIPTION
**Pull request by kathanner**

### CODE REVIEW

**Comment by kathanner** [comment]
@mhigbyflowers -- one of the checks didn't pass, but that's only due to my use of !important in the CSS. I don't see an issue, do you?

**Comment by mhigbyflowers** [comment]
@kathanner I'm ok with !important. I get the test's reasoning but I think there are definitely useful exceptions.

File: public-ui/src/app/views/db-header/db-header.component.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { Component, Input, OnInit } from '@angular/core';
+import { Component, ElementRef, Input, OnInit, Renderer2 } from '@angular/core';
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/db-header/db-header.component.ts:1]: ElementRef is used when selecting elements

File: public-ui/src/app/views/db-header/db-header.component.ts [hunk 2]
```diff
@@ -7,20 +8,39 @@ import { environment } from '../../../environments/environment';
...
-  constructor() {}
+  constructor(private el: ElementRef, private renderer: Renderer2) { }
 
   ngOnInit() {
     this.allOfUsUrl = environment.researchAllOfUsUrl;
   }
 
-  public toggleTopMenu() {
-    this.openTopMenu = !this.openTopMenu;
+  public clickMobileNav() {
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/db-header/db-header.component.ts:23]: click events are handled in the template. see: db-header.component.html

File: public-ui/src/app/views/db-header/db-header.component.ts [hunk 3]
```diff
@@ -7,20 +8,39 @@ import { environment } from '../../../environments/environment';
...
+    for (let i = 0; i < mobileNavAcc.length; i++) {
+      mobileNavAcc[i].addEventListener('click', function () {
+        const nextup = this.lastElementChild;
+        nextup.classList.toggle('mobileopen');
+        this.classList.toggle('flip');
+      });
+    }
+  }
+
+  public headerSearch(f: NgForm) {
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/db-header/db-header.component.ts:43]: I went ahead and hooked up the search form

File: public-ui/src/app/views/db-header/db-header.component.ts [hunk 4]
```diff
@@ -1,4 +1,5 @@
-import { Component, Input, OnInit } from '@angular/core';
+import { Component, ElementRef, Input, OnInit, Renderer2 } from '@angular/core';
+import { NgForm } from '@angular/forms';
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/db-header/db-header.component.ts:2]: ngForm allows simple passing of form values

File: public-ui/src/app/views/db-header/db-header.component.html [hunk 5]
```diff
@@ -1,177 +1,74 @@
...
+					</div>
+				</div>
+			</div>
+			</div><!-- .site-branding-container -->
+			<div class="site-nav-container">
+				<div class="site-branding-nav">
+					<nav id="site-navigation" class="main-navigation">
+					<div class="menu-primary-container">
+						<ul id="menu-primary" class="main-menu">
+							<li (click)="clickMobileNavItem($event)" class="menu-item menu-item-has-children">
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/db-header/db-header.component.html:36]: example of a click event in the template

**Review by mhigbyflowers — CHANGES_REQUESTED** [review]
I can't approve this PR as it is. When I look at this branch the CSS looks broken 
image

**Pull request closed without merging**